website/src/components/Submission/DataUploadForm.tsx

```diff
@@ -216,11 +216,28 @@ const InnerDataUploadForm = ({
                             <a href='/docs/concepts/metadataformat' className='text-primary-700 opacity-90'>
                                 metadata format
                             </a>
-                            . You can download{' '}
-                            <a href={routes.metadataTemplate(organism, action)} className='text-primary-700 opacity-90'>
-                                a template
+                            . You can download a{' '}
+                            <a
+                                href={routes.metadataTemplate(organism, action, 'tsv')}
+                                className='text-primary-700 opacity-90'
+                            >
+                                TSV
+                            </a>
+                            {', '}
+                            <a
+                                href={routes.metadataTemplate(organism, action, 'xls')}
+                                className='text-primary-700 opacity-90'
+                            >
+                                XLS
```

Comment:
I'm sorry to indicate this now, but my take would be that we shouldn't provide an XLS template. XLS was deprecated in 2007! It adds to the screen and the load on users for no real purpose (some won't know what to choose and then will likely get a message about XLS being deprecated).

Reply:
(by contrast, supporting it for upload seems fine - no downside except for the need to maintain it - which I'm fine with for now)

Reply:
makes a lot of sense! I was on autopilot just doing this, but I think you're right. How much of the XLS relates stuff should I get rid of? Maybe just all of the stuff in this PR? (I.e. also the functionality to even get it by API)

Reply:
Yeah my take would be to remove all the template-related functionality

Reply:
(thanks!)

Reply:
done!